db/models.py

```diff
@@ -50,3 +54,65 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
```

Comment:
Read documentation. Import `settings` form `django.conf`

[Link](https://docs.djangoproject.com/en/5.1/topics/settings/#using-settings-in-python-code)